src/Resources/config/services/controller.xml

```diff
@@ -9,6 +9,8 @@
             <argument type="service" id="sylius.repository.taxon"/>
             <argument type="service" id="sylius.context.locale"/>
             <argument type="service" id="event_dispatcher"/>
+            <argument>%env(ALGOLIA_APP_ID)%</argument>
```

Comment:
Should be parameters, not env vars. The parameters should be made from configuration. Then you can use env vars to populate your parameters

Reply:
i.e. `setono_sylius_algolia.app_id` and `setono_sylius_algolia.search_api_key`